Header enum_header
string name
location[] points